### ISSUE TITLE
a11y: Enable CI gate to not add any more a11y violations

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#19117]
**Is your feature request related to a problem? Please describe.**

Enable CI gate to not add any more a11y violations

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
